source/psp/video_hardware_model.cpp

```diff
@@ -561,19 +561,19 @@ void Mod_LoadTextures (lump_t *l)
 		if (loadmodel->bspversion == HL_BSPVERSION || loadmodel->bspversion == NZP_BSPVERSION)
 		{
 			
-			char filename[64];		// Filename to check r4w file
-			byte *f;
-			sprintf(filename, "textures/maps/%s/%s.r4w", sv.name, mt->name);		// search in textures/maps/MAPNAME/TEXNAME
+			// char filename[64];		// Filename to check r4w file
+			// byte *f;
+			// sprintf(filename, "textures/maps/%s/%s.r4w", sv.name, mt->name);		// search in textures/maps/MAPNAME/TEXNAME
 			
-			f = static_cast<byte*>(COM_LoadHunkFile(filename));
+			// f = static_cast<byte*>(COM_LoadHunkFile(filename));
 			
-			if (!f) {
-				sprintf(filename, "textures/%s.r4w", mt->name);					// search in textures/TEXNAME
-				f = static_cast<byte*>(COM_LoadHunkFile(filename));
-			}
+			// if (!f) {
+			// 	sprintf(filename, "textures/%s.r4w", mt->name);					// search in textures/TEXNAME
+			// 	f = static_cast<byte*>(COM_LoadHunkFile(filename));
+			// }
```

Comment:
r4w support should probably never come back, there's no official tools nor documentation, and the new clut4 code isn't compatible, probably.